solr/core/src/java/org/apache/solr/response/transform/DocTransformer.java

```diff
@@ -93,6 +107,12 @@ public void setContext( ResultContext context ) {
   public String[] getExtraRequestFields() {
     return null;
   }
+
+  /**
+   * This is called after a transformer has been applied to all the documents in the results set
+   */
+  public void finish(){
```

Comment:
is it possible to assert that finish is called always during the tests on the base class?